app/views/users/_show.html.erb

```diff
@@ -0,0 +1,7 @@
+<li class="post-item">
+    <h3>Post#<%= post.id %></h3>
+    <div class="inner-items">
+        <p><%= post.text %></p>
+        <p class="item-no">Comments: <%= post.comments_count == nil ? 0 : post.comments_count %>, Likes: <%= post.likes_count == nil ? 0 : post.likes_count %></p>
```

Comment:
- [ ] Let's use the most recent posts model method you created in the previous milestone, this method should render only the three most recent posts for a user.